examples/combined-openapi-mock/varkes_config.json

```diff
@@ -6,7 +6,8 @@
             "specification": "apis/schools.yaml",
             "name": "Schools API",
             "auth": "basic",
-            "persistence": true
+            "persistence": true,
+            "registerSpec": true

```

Comment:
please remove the "registerSpec" everywhere in the examples

Reply:
but then I need to remove it from the fillServiceMetadata function, because if I remove them then it will be treated as false

Reply:
I thought I implemented to have true as default, please double-check

Reply:
yeah that's true..I have tried it before but didn't work...maybe it was a different error

Reply:
removed them now

Reply:
still there

Reply:
I just pushed now